modules/issue_tracker/jsx/IssueForm.js

```diff
@@ -338,7 +338,7 @@ class IssueForm extends Component {
         console.error(err);
         this.setState({submissionResult: 'error'});
         let msgType = 'error';
-        let message = err.responseJSON.message || 'Failed to submit issue :(';
+        let message = err.responseText.message || 'Failed to submit issue :(';
```

Comment:
I don't think this is correct.. most of the ajax calls return a JSON  object with a "message" key. If there's any that don't, it would probably make more sense to update those